src/test/codegen/issue-73827-bounds-check-index-in-subexpr.rs

```diff
@@ -0,0 +1,18 @@
+// This test checks that bounds checks are elided when
+// index is part of a (x | y) < C style condition
+
+// min-llvm-version: 11.0.0
+// compile-flags: -O
+
+#![crate_type = "lib"]
+
+// CHECK-LABEL: @get
+#[no_mangle]
+pub fn get(array: &[u8; 8], x: usize, y: usize) -> u8 {
+    if x > 7 || y > 7 {
+        0
+    } else {
+        // CHECK-NOT: panic_bounds_check
```

Comment:
I'd prefer to move this line up and move it below the CHECK-LABEL.

Reply:
I don't want to write this as a dismissal, but I personally prefer having them where I expect the instruction to (not) be.

Reply:
That is fine. I just prefer to do whatever <https://llvm.org/docs/CommandGuide/FileCheck.html> does.

Reply:
I guess when you have two languages that are close enough, mixing them could seem messy. But with Rust I feel marking the problematic point can have a tiny bit of usefulness :)